src/employee/quiz.md

```diff
@@ -4,33 +4,36 @@ Welcome at VI! To get you going we want to check your know-how. So let's start w
 > Goal: We know where you stand and how to proceed with your work at VI.
 
 ---
+## Development Basics
 ### Git
 
 <questions-component>
   <question-component>What's Git?</question-component>
-  <answer-component>Git is a versioning software which allows developers to manage different versions of their code. It is like a time machine, which allows you to go back in time and merge different timelines together.</answer-component>
+  <answer-component>Git is a versioning software which allows users to manage different versions of their code. It is like a time machine, which allows you to go back in time and merge different timelines together.</answer-component>
+  <question-component>What's the difference between Git and GitHub?</question-component>
+  <answer-component>Git is the tool, GitHub is the platform. On GitHub you can host repositories.</answer-component>
+  <question-component>What's the Git Flow?</question-component>
+  <answer-component>Git Flow is a branching model. It serves blueprints on how to name branches and how to handle merging them together.</answer-component>
```

Comment:
Same for this question. The fundamental questions like what's a branch and what does merge mean, are dealt with later in the chapter.

Maybe we can reorder the questions.